docker/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:9.5-slim
 
+ARG rapture_repo_suffix=20181011-3
```

Comment:
Can we keep this as "repo_suffix"? I'm not sure anyone understands what rapture means except for us.

Reply:
Done.